game/src/test/scala/io/aigar/game/CellSpec.scala

```diff
@@ -63,5 +63,21 @@ class CellSpec extends FlatSpec with Matchers {
 
     cell.position should equal(new Vector2(42f, 42f))
   }
+
+  it should "Be in the cell" in {
```

Comment:
Could we update the test description to make it more explicit? :)

E.g. 
`contain (42,42) with a mass of 100`